### DISCUSSION TITLE
Is there a way to disable remote desktop access on cleints? I already have a screen sharing solution and having desktop remote access is a pivacy and security risk.

### DISCUSSION THREAD

**Discussion by grahamIT** [Ylianst/MeshCentral#5729] [Q&A]
I'm looking to disable the desktop access as its not necessary and intrusive.

**Comment by silversword411** [comment]
Have you looked at the schema file to see valid json options?
https://github.com/Ylianst/MeshCentral/blob/57f77057b4210a3787c263b803906b99c4c09d31/meshcentral-config-schema.json#L1709
